### PR TITLE
fix: add `wrong` class to image popup 

### PR DESCRIPTION
**Pull request by js87zz**
### Please check if the PR fulfills these requirements
- [x] It's the right issue type on the title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has a description of the breaking change

### Description
* added `wrong` class to image popup 
**link**
image
**image**
image
* fixed that cannot toggle italic command
image
* fixed losing the focus after executing the `color` command

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: plugins/color-syntax/src/index.ts [hunk 1]
```diff
@@ -60,11 +80,22 @@ export default function colorSyntaxPlugin(
   const colorPicker = ColorPicker.create(colorPickerOption);
   const button = createApplyButton(i18n.get('OK'));
 
-  button.addEventListener('click', () => {
-    const selectedColor = colorPicker.getColor();
+  eventEmitter.listen('focus', (editType) => {
+    const containerClassName =
+      editType === 'markdown' ? `${PREFIX}md-container` : `${PREFIX}ww-container`;
```
Comment (jungeun-cho) [inline plugins/color-syntax/src/index.ts:85]: (개인적인 취향차이지만) 한줄에 작성하는건 어떻할까요?~
```js
const containerClassName = `${PREFIX}${editType === 'markdown' ? 'md' : 'ww'}-container`;
```

File: apps/editor/src/ui/components/toolbar/imagePopupBody.ts [hunk 2]
```diff
@@ -21,46 +23,72 @@ interface Props {
...
 
-      if (files?.length) {
-        const imageFile = files.item(0)!;
-        const hookCallback: HookCallback = (url, text) =>
-          this.props.execCommand('addImage', { imageUrl: url, altText: text || altText });
+  private emitAddImage() {
+    const imageUrlEl = this.refs.url as HTMLInputElement;
+    const altTextEl = this.refs.altText as HTMLInputElement;
+    const imageUrl = imageUrlEl.value;
+    const altText = altTextEl.value || 'image';
```
Comment (jungeun-cho) [inline apps/editor/src/ui/components/toolbar/imagePopupBody.ts:73]: `altTextEl.value ?? 'image'` 안전하지 않을까합니다.

**Review by jungeun-cho — APPROVED** [review]
리뷰완료했습니다!

File: apps/editor/src/ui/components/toolbar/imagePopupBody.ts (hunk 2 above)
Comment (js87zz) [reply apps/editor/src/ui/components/toolbar/imagePopupBody.ts:73]: 빈 문자열인 경우에도 적용이 되어야 해서 유지하겠습니다~!

**Pull request merged**